cfgov/v1/migrations/0226_update_enforcement_action_data.py

```diff
@@ -0,0 +1,42 @@
+# Generated by Django 2.2.13 on 2020-07-13 17:19
+from django.db import migrations
+from v1.models.learn_page import EnforcementActionStatus
+from v1.util.ref import enforcement_statuses
+
+
+def get_value(label):
+    for status in enforcement_statuses:
+        if label == status[1]:
+            return status[0]
+
+
+def forward(apps, schema_editor):
+    for enforcement_status in EnforcementActionStatus.objects.all():
```

Comment:
Per the documentation [here](https://docs.djangoproject.com/en/3.0/topics/migrations/#data-migrations), best practice when accessing Django model classes in a migration is to use something like this:

```py
EnforcementActionStatus = apps.get_model('v1', 'EnforcementActionStatus')
```

instead of importing at the top of the module as you do here. The reason for this is that you want to make sure that the code is accessing the "historical" model state at the time this migration was written.

Reply:
Gooood to know. Thanks for the explanation.

In that case, would it also make sense to remove the reference to `v1.util.ref.enforcement_statuses`? I could just... make a dictionary that's local to this file that's a mapping of the values I want to update?